common/api/bity.js

```diff
@@ -0,0 +1,82 @@
+import axios from 'axios';
+
+// https://stackoverflow.com/questions/9828684/how-to-get-all-arguments-of-a-callback-function
+export function combineAndUpper() {
+    let args = [];
+    let newString = '';
+    for (let i = 0; i < arguments.length; ++i) args[i] = arguments[i];
+    args.forEach((each) => {
+        newString = newString.concat(each.toUpperCase())
+    });
+    return newString
+}
+
+export default class Bity {
+    constructor() {
+        this.SERVERURL = 'https://myetherapi.com';
```

Comment:
even if we want instances with different urls, defaults belong to separate files.